s3-b2bi-s3/README.md

```diff
@@ -0,0 +1,103 @@
+
+Important: this application uses various AWS services and there are costs associated with these services after the Free Tier usage - please see the [AWS Pricing page](https://aws.amazon.com/pricing/) for details. You are responsible for any AWS costs incurred. No warranty is implied in this example.
+
+## Requirements
+
+- [Create an AWS account](https://portal.aws.amazon.com/gp/aws/developer/registration/index.html) if you do not already have one and log in. The IAM user that you use must have sufficient permissions to make necessary AWS service calls and manage AWS resources.
+- [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html) installed and configured
+- [Git Installed](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+- [AWS Serverless Application Model](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html) (AWS SAM) installed
+
+
+## Deployment Instructions
+
+1. Create a new directory, navigate to that directory in a terminal and clone the GitHub repository:
+   ```bash
+   git clone https://github.com/aws-samples/serverless-patterns
+   ```
+2. Change directory to the pattern directory:
+   ```bash
+   cd s3-b2bi-s3
+   ```
+3. From the command line, use AWS SAM to deploy the AWS resources for the pattern as specified in the template.yml file:
+   ```bash
+   sam deploy -g -t template1.yaml
+   ```
+4. During the prompts:
+
+   - Enter a stack name. I am assuming stack name to be `b2bi-stack1`.
+   - Enter the desired AWS Region. Please refer to the documentation for the list of supported regions.
+   - Enter `BusinessName`, `Email` and `Phone`.
+
+5. Note the outputs from the SAM deployment process. These contain the resource names and/or Ids which are used for next step as well as testing.
+
+6. Log into [AWS B2B Data Interchange](https://us-east-1.console.aws.amazon.com/b2bi/home). Click on the Transformers tab on the left hand navigation menu. Click on the `MyTransformer`. Click on the `Set Status` button and select `Active`. Please ensure that the `MyTransformer` status is set to `Active`. 
+
+7. From the command line, use AWS SAM to deploy the AWS resources for the pattern as specified in the template.yml file:
+   ```bash
+   sam deploy -g -t template2.yaml
+   ```
+8. During the prompts:
+
+   - Enter a stack name. I am assuming stack name to be `b2bi-stack2`.
+   - Enter the desired AWS Region. Please refer to the documentation for the list of supported regions.
+   - Enter `BusinessName`, `Email`, `Phone`, `EDI214InputBucketName`, `EDI214OutputBucketName`, `MyProfileId` and `MyTransformerId`. Please note the last four parameter values should be given as per the output from the `b2bi-stack1` output. 
+
+   Once you have run `sam deploy --guided` mode once and saved arguments to a configuration file (samconfig.toml), you can use `sam deploy` in future to use these defaults.
```

Comment:
I don't understand how this relevant here. Could you elaborate please?

Reply:
Updated. Please let me know if anything else to change.